torchtext/experimental/vectors.py

```diff
@@ -111,7 +112,7 @@ def FastText(language="en", unk_tensor=None, root=".data", validate_file=True):
         checksum = CHECKSUMS_FAST_TEXT.get(url, None)
 
     downloaded_file_path = download_from_url(url, root=root, hash_value=checksum)
-    tokens, vectors, dup_tokens = _load_token_and_vectors_from_file(downloaded_file_path)
+    tokens, vectors, dup_tokens = torch.ops.torchtext._load_token_and_vectors_from_file(downloaded_file_path, ord(' '), 5)
```

Comment:
Why is using "ord" necessary?

Reply:
Spoke to the torchscript team and `char` isn't a supported type. Also thought using a string would be too hacky as there is the potential for the user to pass in multiple characters

Reply:
In theory a delimiters of multiple characters isn't necessarily bad, right? I think it's more general to pass a string and then error out if it has more than length 1, so that in the future we could extend support to more complex delimiters instead of inherently limiting ourselves to one character.